index.html

```diff
@@ -8,6 +8,57 @@
     <title>Document</title>
 </head>
 <body>
-    
+    <!-- -----header----- -->
+    <div class="container-header-bg">
+        <div class="header-logo-menu">
+            <div class="wrapper-img-logo">
+                <img class="img-logo" src="images/logoKidz.png" alt="logo">
+            </div>
+            <div class="header-contacts-and-menu">
+                <div class="header-contacts-register-login">
+                    <div class="header-email-phone">
+                        <a href="tel:18006333469"><i class="fas fa-phone text-white contacts"></i> 1-800-633-3469 </a>
+                        <a href="mailto:mailedukado.com"><i class="fas fa-envelope text-white"></i> mailedukado.com </a>
+                    </div>
+                    <div class="header-search-register-login">
+                        <a href="#"><i class="fas fa-search color-for-icons"></i></a>
+                        <a href="#"><i class="fas fa-user color-for-icons"></i> Register</a>
+                        <a href="#"><i class="fas fa-sign-in-alt color-for-icons"></i> Login</a>
+                    </div>
+                </div>
+                <div class="line"></div>
+                <div class="header-menu">
+                    <nav class="main-nav">
+                        <ul>
+                            <li><a href="#">Home</a></li>
+                            <li><a href="#">Academic</a></li>
+                            <li><a href="#">Admission</a></li>
+                            <li><a href="#">Teachers</a></li>
+                            <li><a href="#">About</a></li>
+                            <li><a href="#">Blog</a></li>
+                            <li><a href="#">Contact</a></li>
+                        </ul>
+                    </nav> 
+                </div>
+            </div>
+        </div>
+    <!-- -----main-page----- -->
+        <div class="main-page-slider-and-text">
+            <div class="main-page-slider">
+                <div class="wrapper-img-slider">
```

Comment:
You don't need this div, all styles from this div you can write in `main-page-slider` div